util.py

```diff
@@ -691,7 +691,20 @@ def setup_lbaas_http_test_server(backend, ssl=False):
     '''))
 
 
-def setup_lbaas_backend(backend, backend_network, ssl=False):
+def setup_lbaas_udp_test_server(backend):
```

Comment:
I think it might be nice to pass in the message, so the message is built by the tests, and then verified there as well (the message would be defined there, and could then be verified with the same variable). This would also make it a bit clearer.

Reply:
Ok, fair point.